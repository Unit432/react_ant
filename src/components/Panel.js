import React from 'react'
import { Layout } from 'antd'
import DownloadsContainer from '../containers/Downloads'
import Settings from './Settings'
import { Route } from 'react-router-dom'
const { Content } = Layout

class Panel extends React.Component {
  render () {
    return (
      <Content
        style={{
          margin: '24px 16px',
          padding: 24,
          background: '#fff',
          minHeight: 280
        }}
      >
        <Route exact path="/" component={DownloadsContainer} />
        <Route path="/settings" component={Settings} />
      </Content>
    )
  }
}

export default Panel
