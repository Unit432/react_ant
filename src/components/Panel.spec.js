import React from 'react'
import Panel from './Panel'
import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom'

describe("Panel", () => {
  it('Panel renders a list of downloads', () => {
    const rendered_panel = renderer
      .create(<MemoryRouter><Panel /></MemoryRouter>)
      .toJSON();
    expect(rendered_panel).toMatchSnapshot()
  })
})
