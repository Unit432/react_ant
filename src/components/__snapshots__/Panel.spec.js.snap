// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Panel Panel renders a list of downloads 1`] = `
<div
  className="ant-layout-content"
  style={
    Object {
      "background": "#fff",
      "margin": "24px 16px",
      "minHeight": 280,
      "padding": 24,
    }
  }
>
  <h1>
    Downloads
  </h1>
</div>
`;
